doc/nrf/protocols/matter/index.rst

```diff
@@ -23,7 +23,9 @@ For a full list of |NCS| and Matter versions, view the following table:
    +--------------------------+-----------------------------------------------------+------------------------+
    | nRF Connect SDK version  | Matter specification version                        | Matter SDK version     |
    +==========================+=====================================================+========================+
-   | |release|                | :ref:`1.4.0 <ug_matter_overview_dev_model_support>` | 1.4.0.0                |
+   | v2.9.99 (latest)         | :ref:`1.4.0 <ug_matter_overview_dev_model_support>` | 1.4.0.0                |
+   +--------------------------+                                                     |                        |
+   | v2.9.0-nRF54H20-1-rc3    |                                                     |                        |
```

Comment:
Known issue page needs to be orphaned and the path for KI should be added back here: doc/nrf/releases_and_maturity.rst.

Reply:
Why would we orphan the known issue page? this is a post release PR

Reply:
My bad, I meant unorphaned.